### ISSUE TITLE
two questions about code

### ISSUE THREAD

**Issue by nodoth1sl** [sicara/easy-few-shot-learning#22]
Hello! First of all, thank you very much for your code! I have a few questions for you.

The first question is: I want my model to accurately classify my own pictures. Must the test set and training set be mutually exclusive? Because I have fewer categories, only 6.

The second question is: how can I change your code so that I can run it myself?

Thank you sincerely!

**Comment by nodoth1sl** [comment]
Moreover, why is the accuracy of my model after training not as high as that before training?(use my own images)

**Comment by ebennequin** [comment]
Hi! Thanks you for your kind words.

> The first question is: I want my model to accurately classify my own pictures. Must the test set and training set be mutually exclusive? Because I have fewer categories, only 6.

It's good practice to make training and test set mutually exclusive. When evaluating a Few-Shot Learning algorithm, we usually split the classes between train and test so that we're sure that it's evaluated in a few-shot setting. But it's not a technical requirement, you can perfectly have the train and test set sharing the same classes!

> The second question is: how can I change your code so that I can run it myself?

I don't understand, are you unable to run your code?

> Moreover, why is the accuracy of my model after training not as high as that before training?(use my own images)

Could you please share your training script? Did you start from a pretrained model or a model with random initialization?
